sections/security/login-rate-limit.md

```diff
@@ -1,51 +1,9 @@
-# Implement express rate limiting for login routes
+# Prevent brute-force attacks against authorization

```

Comment:
It would make sense to also update the bullet title 6.12 on README.md to match this, so they are consistent

Reply:
Done